repo2docker/buildpacks/julia/julia_semver.py

```diff
@@ -0,0 +1,119 @@
+# This file implements the julia-specific logic for handling SemVer (Semantic
```

Comment:
Let's convert to a Python file docstring `"""` instead of "#" comments.

Reply:
I believe @betatim  wanted to go over this file once it is merged and change a couple of things into a more Pythonic style. If this is a blocking issue, maybe @betatim or @NHDaly could open a PR against this branch here with such changes?

I just copied this file over from @NHDaly's branch and probably won't have the time to do much more work on this particular code.

Reply:
The change here would be
```
"""This module does something with semver"""
```
instead of `# This module does something with semver`
let's do it here. We should try and stick with "don't push tidying up to tomorrow" because we never do :-/